yarn-project/world-state/src/merkle-tree/merkle_tree_operations_facade.ts

```diff
@@ -18,6 +18,15 @@ export class MerkleTreeOperationsFacade implements MerkleTreeOperations {
     return this.trees.getTreeInfo(treeId, this.includeUncommitted);
   }
 
+  /**
+   * Get the current roots of the commitment trees.
+   * @param includeUncommitted - Indicates whether to include uncommitted data.
```

Comment:
Should the param be shown here when it cannot be passed in, reads like a stale comment